### PR TITLE
Add docs for SAC

### PR DESCRIPTION
**Pull request by yeukfu**
This should close https://github.com/rlworkgroup/garage/issues/2008.

### CODE REVIEW

File: docs/user/algo_sac.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+a measure of randomness in the policy. This has a close connection to the
+exploration-exploitation trade-off: increasing entropy results in more
+exploration, which can accelerate learning later on. It can also prevent the
+policy from prematurely converging to a bad local optimum.
+
+## Default Parameters
+
+```python
+gradient_steps_per_itr=1000
+max_episode_length_eval=1000
```
Comment (avnishn) [inline docs/user/algo_sac.md:37]: is this really the default? I thought that the default was `None`

**Review by avnishn — APPROVED** [review]

File: docs/user/algo_sac.md (hunk 1 above)
Comment (yeukfu) [reply docs/user/algo_sac.md:37]: Sorry. I made a mistake here

File: docs/user/algo_sac.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+     - Continuous
+   * - **Paper**
+     - Soft Actor-Critic Algorithms and Applications :cite:`haarnoja2018soft`
+   * - **Framework(s)**
+     - .. figure:: ./images/pytorch.png
+        :scale: 10%
+
+        PyTorch
+   * - **API Reference**
+     - `garage.torch.algos.BC <../_autoapi/garage/torch/algos/index.html#garage.torch.algos.SAC>`_
```
Comment (ryanjulian) [inline docs/user/algo_sac.md:19]:
```suggestion
     - `garage.torch.algos.SAC <../_autoapi/garage/torch/algos/index.html#garage.torch.algos.SAC>`_
```

File: docs/user/algo_sac.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+   * - **Code**
+     - `garage/torch/algos/bc.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/sac.py>`_
```
Comment (ryanjulian) [inline docs/user/algo_sac.md:21]:
```suggestion
     - `garage/torch/algos/sac.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/sac.py>`_
```

**Review by ryanjulian — APPROVED** [review]

**Pull request merged**